PaddleNLP/sentiment_classification/run.sh

```diff
@@ -21,7 +21,7 @@ train() {
         --do_train true \
         --do_val false \
         --do_infer false \
-        --model_type ${MODEL_TYPE} \            
+        --model_type $MODEL_TYPE \
```

Comment:
Why removing this `{}` ?  I notice other variables like `${DATA_PATH}` all have this `{}`

Reply:
`${MODEL_TYPE}` is equal to `$MODEL_TYPE`